app/routes/enrich-request.js

```diff
@@ -84,21 +85,32 @@ module.exports = [{
       }
 
       const user = getUser(request)
-
+      const { paymentRequestId, schemeId, frn } = paymentRequest
       await saveDebt({
-        paymentRequestId: paymentRequest.paymentRequestId,
-        schemeId: paymentRequest.schemeId,
-        frn: paymentRequest.frn,
+        paymentRequestId: paymentRequestId,
+        schemeId: schemeId,
+        frn: frn,
         debtType: payload['debt-type'],
         recoveryDate: `${day}/${month}/${year}`,
         createdDate: new Date(),
         createdBy: user.username,
         createdById: user.userId
       })
 
-      await updateQualityChecksStatus(paymentRequest.paymentRequestId, PENDING)
-      await sendEnrichRequestEvent(paymentRequest, user)
+      const isAwaitingManualLedgerDebtData = await db.qualityCheck.findOne({
```

Comment:
Move this into a new module within `quality-check`, refer to it in its `index.js` and import it this way

Reply:
Done